src/legacy/core_plugins/embeddable_api/public/panel/_embeddable_panel.scss

```diff
@@ -0,0 +1,198 @@
+/**
+  * EDITING MODE
+  * Use .dshLayout--editing to target editing state because
+  * .embPanel--editing doesn't get updating without a hard refresh
+  */
+
+$dshEditingModeHoverColor: transparentize($euiColorWarning, lightOrDarkTheme(.9, .7));
+
+.embPanel {
+  z-index: auto;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  position: relative;
+
+  // SASSTODO: The inheritence factor stemming from embeddables makes this class hard to change
+  .panel-content {
+    display: flex;
+    flex: 1 1 100%;
+    min-height: 0; // Absolute must for Firefox to scroll contents
+  }
+
+  .embeddable-root {
+    display: flex;
+    flex: 1 1 100%;
+    height: auto;
+    z-index: 1;
+    min-height: 0; // Absolute must for Firefox to scroll contents 
+  }
+
+  // SASSTODO: Pretty sure this doesn't do anything since the flex-basis 100%,
+  // but it MIGHT be fixing IE
+  .panel-content--fullWidth {
+    width: 100%;
+  }
+
+  .panel-content-isLoading {
+    // completely center the loading indicator
+    justify-content: center;
+    align-items: center;
+  }
+
+  /**
+   * 1. We want the kbnDocTable__container to scroll only when embedded in a dashboard panel
+   * 2. Fix overflow of vis's specifically for inside dashboard panels, lets the panel decide the overflow
+   * 3. Force a better looking scrollbar
+   */
+  .kbnDocTable__container {
+    @include euiScrollBar; /* 3 */
+    flex: 1 1 0; /* 1 */
+    overflow: auto; /* 1 */
+  }
+
+  .visualization {
+    @include euiScrollBar; /* 3 */
+  }
+
+  .visualization .visChart__container {
+    overflow: visible; /* 2 */
+  }
+
+  .visLegend__toggle {
+    border-bottom-right-radius: 0;
+    border-top-left-radius: 0;
+  }
+}
+
+.embPanel--editing {
+  border-style: dashed;
+  border-color: $euiColorMediumShade;
+  transition: all $euiAnimSpeedFast $euiAnimSlightResistance;
+
+  &:hover,
+  &:focus {
+    @include euiSlightShadowHover;
+  }
+}
+
+// LAYOUT MODES
+
+// Adjust borders/etc... for non-spaced out and expanded panels
+.embPanel-withoutMargins,
+.dshDashboardGrid__item--expanded {
+  .embPanel {
+    box-shadow: none;
+    border-radius: 0;
+  }
+}
+
+// Remove border color unless in editing mode
+.embPanel-withoutMargins:not(.embPanel--editing),
+.dshDashboardGrid__item--expanded {
+  .embPanel {
```

Comment:
I messed up the names here, it should be 
```
.dshLayout-withoutMargins:not(.dshLayout--editing),
.dshDashboardGrid__item--expanded {
  .embPanel {
```

so I think i actually need to keep this tree... otherwise it doesn't work.